app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardFragment.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.app.topic.conceptcard
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import kotlinx.android.synthetic.main.conceptcard_fragment.*
+import kotlinx.android.synthetic.main.conceptcard_fragment.view.*
+import org.oppia.app.R
+import org.oppia.app.fragment.InjectableDialogFragment
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+class ConceptCardFragment : InjectableDialogFragment() {
```

Comment:
Please add documentation for this fragment to describe what it is/how it should be used.